src/TwigComponent/src/Resources/doc/index.rst

```diff
@@ -377,6 +410,135 @@ you can store its result on a private property:
           }
       }
 
+Component Attributes
+--------------------
```

Comment:
We probably need something in the Live component docs. As it stands now, I think you'd need `{{ init_live_component(this) }} {{ attributes }}` on the root element, right?

Reply:
Correct, as discussed in slack, I'll add the system to append the live stuff to these attributes in this PR.

Reply:
This is not available in this PR. One neat thing is the live component doesn't need the trait for this to work. If it has the trait, the live attributes are merged. `<div {{ attributes }}>` always works.

Question is, should I make the attributes always available in non-live component templates (this is possible too)? It wouldn't contain any data in this case but we'd basically be able to say: "If you want to be able to pass attributes when mounting your component, add the trait to your component".

Reply:
Wow, that's cool!

> Question is, should I make the attributes always available in non-live component templates (this is possible too)?

I say yes! It would make converting to a live component easier, because you'd already have the `{{ attributes }}`.